src/main/java/core/basesyntax/UserInterface.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class UserInterface {
+    Scanner scanner = new Scanner(System.in);
+
+    public void userInput() {
+        String inputStr;
+        inputStr = scanner.nextLine();
+        List<String> splitInputStr = new ArrayList<>(Arrays.asList(inputStr.split(" ")));
+        FileEditor fileEditor = new FileEditor();
+        while (!splitInputStr.get(0).equals("exit")) {
+            switch (splitInputStr.get(0)) {
```

Comment:
just create case for "exit" with System.exit(0) method which will stop your program